### PR TITLE
Update Release_Build_Creation.rst

### PR DESCRIPTION
**Pull request by justinzw**
Added link to CentOS ISO and tested by section.

Signed-off-by: Justin Woo <justin.woo@seagate.com>

### CODE REVIEW

File: doc/Release_Build_Creation.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ This file consists of the procedure that should be followed to generate the rele
 Procedure
 ***************
 
-#. Setup a CentOS 7.8.2003 system.
+#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7/isos/x86_64/>`_ system.
```
Comment (shailesh-vaidya) [inline doc/Release_Build_Creation.rst:11]: @justinzw Above links shows to `CentOS 7.9.2009` ISO files . For `CentOS 7.8.2003` - We need to use https://vault.centos.org/7.8.2003/isos/x86_64/

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: doc/Release_Build_Creation.rst (hunk 1 above)
Comment (justinzw) [reply doc/Release_Build_Creation.rst]: Good catch! thanks for seeing that. I updated the link to 7.8.2003.

Thanks @shailesh-vaidya

File: doc/Release_Build_Creation.rst [hunk 2]
```diff
@@ -113,7 +113,7 @@ Procedure
 #. Follow this `Guide <Provision Release Build.md>`_ to run your build.
 
 Tested by:
-
+- May 19, 2021: Justin Woo (justin.woo@seagate.com) on  "CentOS 7.8.2003 x86_64" VM running on a Windows 10 Desktop running VMware Workstation 16 Pro.
```
Comment (mukul-seagate11) [inline doc/Release_Build_Creation.rst:116]: It has to be formatted correctly

Comment (justinzw) [reply doc/Release_Build_Creation.rst]: Thanks @mukul-seagate11 I formatted it like how the rest did.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**